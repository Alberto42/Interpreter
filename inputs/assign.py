x = 5
y = 6
y = 7

z = 3
z = "str"

a = True
b = [1]
c = (1,2)

a = False
b = [2]
c = (3)