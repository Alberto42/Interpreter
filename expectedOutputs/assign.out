Ident "z" = VString "str"
Ident "y" = VInt 7
Ident "x" = VInt 5
Ident "c" = VInt 3
Ident "b" = VList (fromList [VInt 2])
Ident "a" = VBoolean False


